packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -101,21 +94,14 @@ const PersonalDetailsForm = ({
         }
     }, [is_tax_residence_popover_open, is_tin_popover_open]);
 
-    const name_dob_clarification_message = (
-        <Localize
-            i18n_default_text='To avoid delays, enter your <0>name</0> and <0>date of birth</0> exactly as they appear on your identity document.'
-            components={[<strong key={0} />]}
-        />
-    );
-
     return (
         <div className={classNames({ 'account-form__poi-confirm-example': is_qualified_for_idv })}>
-            {(is_qualified_for_idv || is_rendered_for_onfido) && !should_hide_helper_image && (
-                <InlineNoteWithIcon message={name_dob_clarification_message} font_size={isMobile() ? 'xxxs' : 'xs'} />
+            {(is_qualified_for_idv || is_qualified_for_onfido) && !should_hide_helper_image && (
```

Comment:
is_qualified_for_idv || is_qualified_for_onfido this condition is repeated many times. maybe we can create a variable and reuse it